### PR TITLE
Add Synth1 VST / Audio Unit

### PR DESCRIPTION
**Pull request by joshka**

### CODE REVIEW

File: Casks/synth1-au.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'synth1-au' do
+  version '1.13beta8'
+  sha256 '78660848d78c19cdda2597bd050a2ec95fa915cc1a1ab598bde0192a1cbc9a6f'
+
+  url 'http://www.geocities.jp/daichi1969/softsynth/Synth1macau113beta8.zip'
```
Comment (adityadalal924) [inline Casks/synth1-au.rb]: You can use a `#{version.delete('.')}` here

**Comment by adityadalal924** [comment]
Do these two depend on each other? If so, which one is considered the "main" one

File: Casks/synth1-au.rb (hunk 1 above)
Comment (joshka) [inline Casks/synth1-au.rb]: You're right. Will fix later tonight.

**Comment by joshka** [comment]
They don't depend on each other, They are two different types of plugin (Audio Unit / VST).
Some DAWs use VSTs (Cubase, Ableton-live, etc.) some use Audio Units (Apple Logic etc.)
In this case, the plugin is released as two separate zip files on the vendor's homepage.
Some plugins are released as both in the same.

If you want to hold off merging this, I'm working on changing to use two new artifacts called vst_plugin and audiounit_plugin, for the purpose of being able to write a bunch of these quickly. In the same vein as the internet_plugin artifact type.

**Comment by adityadalal924** [comment]
That sounds like a more flexible option.

Open up a PR when you're ready and we'll take a look!

Thanks.

(Closing this for the time being, in that case). Feel free to reopen if you end up not making any core changes and we can merge this.

**Pull request closed without merging**